docs/lab/overview/testing-modalities.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: "Testing Modalities"
+---
+
+By integrating with our API, you will have access to all testing modalities.
+
+Three types are currently supported:
+
+- At-home test kit.
+- At-home phlebotomy visit.
+- Walk-in test visit.
+
+## At-home test kits
+
+This involves sending a kit to a patients home that allows the patient to collect their
+sample specimen and send back to our lab networks in the mail.
+
+At home testing kits are comprised of a number of components such as:
+
+ <div className="flex flex-row">
+  <div>
+
+- Mailer Box
+- Blade Lancets
+- Plasters
+- Gauze
+- ADX Card
+- Blood collection card
+- Saliva Tubes
+- Collection Supplies
+- Instruction Booklet
+
+Additional components can be included in the test kit box, to fully customize the kit, just contact us support@tryvital.io.
+
+  </div>
+  <img src="/img/wyndly.png" className="w-1/2 h-1/2" />
+</div>
+
+## At-home phlebotomy
+
+At-home phlebotomy requires mobile phlebotomists visiting a patient at their home and carrying out the phlebotomy draw. The sample is then delivered to the lab by the phlebotomist.
+
+The flow for at-home phlebotomy orders is as follows:
+
+- You [place an order](/lab/overview/quickstart) and we acknowledge your request.
+- We start processing the order in the background. For at-home phlebotomy, this means generating the requisition form for the lab test.
+- Once the requisition form is ready, we send you a webhook update and we send the patient an email to let them know they can now book an appointment.
+- By following the link in the email, the patient goes through our appointment booking dashboard, where they can select a location and time from the available slots.
+- Once the appointment takes place and the sample is drawn, it will be taken to the lab.
+- Once the lab analyzes the results, the order is complete and you can [fetch the results](/lab/results/result-formats).
```

Comment:
I would likely remove the notion of "you" and "we" from the steps. But feel free to dismiss this comment, since it's more stylistic:

```
- Place an order via 
-The order is processed in background. For at-home phlebotomy, this means generating the requisition form for the lab test.
- Once the requisition form is ready, a webhook update and an email to the patient are sent. The email will them know that they can now book an appointment.
- The patient goes through Vital's appointment booking dashboard, where they can select a location and time from the available slots.
 - Once the appointment takes place and the sample is drawn, it will be taken to the lab.
 - Once the lab analyzes the results, the order is complete and the results can be fetched
```